src/domains/webhook/service/webhookRegistrationStrategy.linear.impl.ts

```diff
@@ -0,0 +1,35 @@
+import type { ProviderWebhookRegistrationData } from '@domains/webhookSource';
+import { createApolloClient } from '@utils/graphql/apollo';
+import { CreateWebhookMutation } from '@utils/graphql/queries';
+import { type WebhookRegistrationStrategy } from '@domains/webhook/service/webhookRegistrationStrategy';
+import { NodeEnv } from '@utils';
+
+export class LinearWebhookRegistrationStrategy implements WebhookRegistrationStrategy {
+  supports(issueProviderName: string): boolean {
+    return issueProviderName.toUpperCase().trim() === 'LINEAR';
+  }
+
+  async registerWebhook(providerApiKey: string, providerProjectId: string, trickerWebhookSourceId: string): Promise<ProviderWebhookRegistrationData> {
+    const apolloClient = createApolloClient(providerApiKey);
+
+    const url = process.env.NODE_ENV === NodeEnv.DEV ? process.env.NGROK_STATIC_DOMAIN! : 'tricker.com.ar';
+
+    const { data: mutationResult } = await apolloClient.mutate({
+      mutation: CreateWebhookMutation,
+      variables: {
+        input: {
+          label: `Tricker Project Integration`,
```

Comment:
You may give a unique identifier to the webhook in linear. This is to avoid problem among developers using dev environment